rocq-runtime.opam.template

```diff
@@ -1,3 +1,11 @@
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml" {>= "4.09.0"}
+  "ocamlfind" {>= "1.8.1" & (>= "1.9.8" | os != "windows") }
```

Comment:
I didn't figure out how to put this in the dune-project so I put it in the .template, if you know how please fix